src/app/modules/epidemiologia/components/ficha-epidemiologica-crud/ficha-epidemiologica-crud.component.ts

```diff
@@ -361,21 +361,24 @@ export class FichaEpidemiologicaCrudComponent implements OnInit, OnChanges {
         }
 
         if (this.fichaPaciente) {
-            this.formEpidemiologiaService.update(this.fichaPaciente._id, fichaFinal).subscribe(
-                () => {
+            try {
+                this.formEpidemiologiaService.update(this.fichaPaciente._id, fichaFinal).subscribe((fichaFinal) => {
```

Comment:
Estas subscripciones tienen implementada una manera de capturar los errores y manejar el resultado del request que nos ahorra el uso de try-catch (por eso vas a ver que no los usamos del lado de la app). Te dejo la sugerencia para este caso..
```suggestion
this.formEpidemiologiaService.update(this.fichaPaciente._id, fichaFinal).subscribe({
            next: (fichaFinal) => {
                    this.plex.toast('success', 'Su ficha fue actualizada correctamente');
                    this.postSave(fichaFinal);
            },
            error: () => this.plex.toast('danger', 'ERROR: La ficha no pudo ser actualizada')
        });
                
```

Reply:
Muchas gracias, quise salvar el error del subscribe deprecado pero lo había entendido mal claramente